packages/help-text/src/helpTextMixin.ts

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
I do this because a `mixin` is not in and of itself a class. I actually do this as kabob case in other uses 😱 https://github.com/adobe/spectrum-web-components/blob/main/packages/shared/src/observe-slot-text.ts should I match that up/

Reply:
I worked the naming here a little bit to be more normative to other mixins in the library.